src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -502,6 +502,11 @@ private function addServiceMethodCalls($id, Definition $definition, $variableNam
     private function addServiceOverriddenMethods($id, Definition $definition)
     {
         $class = $this->container->getReflectionClass($definition->getClass());
+
+        if (!$class) {
+            throw new RuntimeException(sprintf('Unable to configure service "%s": class could not be reflected.', $id));
```

Comment:
`class "%s" not found.` is the only reason why it can fail - and "be reflected" will be obscure to people I guess, so I'm still up for the change